nixos/release.nix

```diff
@@ -92,15 +92,33 @@ let
   }).config));
 
 
+  cleanConfig = import ./lib/eval-config.nix {
```

Comment:
But regular manual evaluation does depend on them. So this slows down nixos-rebuild and can even prevent evaluation from succeeding (e.g. if an option refers to a package that throws an error). And it makes the hash of the manual that Hydra builds different from the one users get (so they will need to build the manual from source).